src/main/java/randoop/test/RegressionCaptureGenerator.java

```diff
@@ -13,12 +13,7 @@
 import randoop.ExecutionOutcome;
 import randoop.NormalExecution;
 import randoop.NotExecuted;
-import randoop.contract.EnumValue;
-import randoop.contract.IsNotNull;
-import randoop.contract.IsNull;
-import randoop.contract.ObjectContract;
-import randoop.contract.ObserverEqValue;
-import randoop.contract.PrimValue;
+import randoop.contract.*;
```

Comment:
Please do not make gratuitous changes to import statements.  The style of Randoop is not to use wildcards.